README.md

```diff
@@ -112,6 +112,8 @@ The main React hook to execute HTTP requests.
   **Returns**
 
   A promise containing the response. If the request is unsuccessful, an exception is thrown and must be handled manually.
+  
+- `manualCancel()` - A function to cancel outstanding requests manually.
```

Comment:
this should go before line 112

Reply:
Moved it before to the bullet point after the 'execute' parameter.